iDiff/differs/differs.go

```diff
@@ -1,86 +1,50 @@
 package differs
 
 import (
-	"bytes"
 	"errors"
-	"os"
-	"reflect"
 
 	"github.com/GoogleCloudPlatform/runtimes-common/iDiff/utils"
 )
 
-var diffs = map[string]func(string, string, bool, bool) (string, error){
-	"hist":    HistoryDiff,
-	"history": HistoryDiff,
-	"file":    FileDiff,
-	"apt":     AptDiff,
-	"linux":   AptDiff,
-	"pip":     PipDiff,
-	"node":    NodeDiff,
+type ImageDiff struct {
```

Comment:
nit: This struct name is a little confusing to me just because ImageDiff makes me think the diff has been done but it hasn't yet... not super important just something to consider